README.md

```diff
@@ -210,6 +210,77 @@ filters=stg,database,data
 
 Then the `lsi -p stg-database` command will be equivalent to the above.
 
+#### Printing JSON output
+
+You can print JSON output with `lsi` by passing through the `--ansible` option with no other ansible specific options. The ansible options include `--playbook` and `--module`. `--module` can take the optional `--params`.
```

Comment:
What does this mean, to print JSON output? And why would this be under the `--ansible` option?